nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -428,14 +462,71 @@ void reportInitErrorOnField(Symbol symbol, VisitorState state, Description.Build
                   FIELD_NO_INIT, "@NonNull static field " + fieldName + " not initialized"),
               tree,
               builder,
-              state));
+              state,
+              symbol));
     } else {
       state.reportMatch(
           createErrorDescription(
               new ErrorMessage(FIELD_NO_INIT, "@NonNull field " + fieldName + " not initialized"),
               tree,
               builder,
-              state));
+              state,
+              symbol));
+    }
+  }
+
+  /**
+   * Suggests a type change of an element in a source code that can resolve the error.
+   *
+   * @param state Visitor state.
+   * @param target Target element to alternate it's type.
+   * @param errorMessage Error caused by the target.
+   */
+  public void suggest(VisitorState state, Symbol target, ErrorMessage errorMessage) {
+    // Skip if element has an explicit @Nonnull annotation.
+    FixSerializationConfig fixSerializationConfig = config.getFixSerializationConfig();
+    if (Nullness.hasNonNullAnnotation(target, config)) {
```

Comment:
How can this ever happen?

Reply:
```suggest``` which is now renamed to ```serializeFixSuggestion``` creates ```SuggestedFixInfo``` for ```target``` symbol. Please see the example below where we have in our test case:

```java
public Object returnNullable()
     return null;
}
```

In the example above ```FixSerialization``` will suggest to make ```returnNullable```, ```@Nullable```.

But in the example below, ```FixSerialization``` will acknowledge the explicit ```@Nonnull``` annotation and will not suggest any type change.

```java
public @Nonnull Object returnNullable()
     return null;
}
```

Reply:
7bad2b3cae3e10263a129e963ab3d2246e122f22